Gemfile.lock

```diff
@@ -156,8 +154,8 @@ DEPENDENCIES
   shoulda-matchers (~> 2.8.0)
   spring
   spring-watcher-listen (~> 2.0.0)
-  sqlite3
+  sqlite3 (~> 1.3.6)
   tzinfo-data
 
 BUNDLED WITH
-   1.13.2
+   2.0.1
```

Comment:
hmmm, too many gems are updated. Is everything works?

Reply:
Yes, it works fine on my end.